core/src/main/java/cz/xtf/core/event/helpers/EventHelper.java

```diff
@@ -0,0 +1,11 @@
+package cz.xtf.core.event.helpers;
+
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class EventHelper {
```

Comment:
maybe EventTimeConverter or EventTimeHelper is better name

Reply:
This is supposed to be helper for anything for`Event` not just in matter of time.

Reply:
I've mentioned it because it's hard to find it and people might use such helper method on their own.

Reply:
However I don't see a merge blocker.

Reply:
Ok. Can you resolve the conversation then? I'd like to keep the name.